cf/data/data.py

```diff
@@ -3235,45 +3236,44 @@ def rechunk(
         balance=False,
         inplace=False,
     ):
-        """Convert blocks in the dask array for new chunks.
-
-        See `dask.array.rechunk`for more details.
+        """Change the chunk struture of the data.
 
-        .. versionadded:: 4.0.0
+        .. versionadded:: TODODASK
 
-        .. seealso:: `chunks`
+        .. seealso:: `chunks`, `dask.array.rechunk`
```

Comment:
I hadn't thought about this before, but tried building a development version of the docs from this branch and such references all work seamlessly with intersphinx to link directly to relevant methods under the Dask docs, which is very neat. I guess you realised this - good thinking!

It might be useful to add similar 'see also' references to relevant Dask methods throughout the Data module where appropriate, if we aren't already doing so already (I've made a mental note to do so).